queries/sflog/highlights.scm

```diff
@@ -0,0 +1,44 @@
+; highlights.scm
+(number) @number
+
+(identifier) @variable
+
+(version) @number
```

Comment:
version is a string, so maybe `@string.special` will fit?

Reply:
Thanks for the help! I'm still learning what @ fits what node best.
Updating it.

Reply:
@lucario387 it seems actions all passed. Pls let me know if any more actions are required on my end :)